_Tests/AudibleApi.Tests/L0/Authorization/PrivateKeyTests.cs

```diff
@@ -46,16 +47,40 @@ public void bad_ending()
 		public void valid_strings()
 		{
 			var justBeginningAndEnd = "-----BEGIN RSA PRIVATE KEY-----END RSA PRIVATE KEY-----";
-			new PrivateKey(justBeginningAndEnd)
-				.Value.Should().Be(justBeginningAndEnd);
+			Assert.ThrowsException<FormatException>(() => new PrivateKey(justBeginningAndEnd)
+				.Value.Should().Be(justBeginningAndEnd));
 
 			var withWhitespace = "\r\n  -----BEGIN RSA PRIVATE KEY-----END RSA PRIVATE KEY-----\r\n  ";
-			new PrivateKey(withWhitespace)
-				.Value.Should().Be(withWhitespace);
+			Assert.ThrowsException<FormatException>(() => new PrivateKey(withWhitespace)
+				.Value.Should().Be(withWhitespace));
 
 			var full = @"
 -----BEGIN RSA PRIVATE KEY-----
-key stuff here
+MIIEpgIBAAKCAQEA5nPbGSVDmlEH2tJa6kz/P2HI8IeirhfPHdmi+X/nsb9i3WNf
```

Comment:
Are you sure you want to include a private key in a public repo?

Reply:
I deleted the device after it was registered. It's useless now.

Reply:
Perfect. I was hoping this was the case.